### ISSUE TITLE
Luma theme meta viewport tag contains maximum-scale=1.0, user-scalable=no

### ISSUE THREAD

**Issue by svossen** [magento/magento2#5403]
The Luma theme viewport meta tag contains `maximum-scale=1.0, user-scalable=no`. This is a really bad practise and is really harmful for the accessibility of the interface! Developers should not decide how a user should interact with the interface. This should be up to the user.

Fortunately the blank theme contains the correct meta viewport tag. But some shops will probably use 
Luma as main theme or as fallback so it's important that those values are removed from the Luma theme as well.
## Steps to reproduce

Always, meta tag is added to the header
## Expected result

`<meta name="viewport" content="width=device-width, initial-scale=1">`
## Actual result

`<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0, user-scalable=no">`

**Comment by hostep** [comment]
There used to be a discussion about this a while ago, see: https://github.com/magento/magento2/pull/1733

**Comment by svossen** [comment]
Missed that one. Thank you.
